mmv1/products/compute/api.yaml

```diff
@@ -15647,6 +15647,99 @@ objects:
         required: true
         update_verb: :POST
         update_url: 'projects/{{project}}/regions/{{region}}/targetHttpsProxies/{{name}}/setUrlMap'
+  - !ruby/object:Api::Resource
+    name: 'RegionTargetTcpProxy'
+    kind: 'compute#targetTcpProxy'
+    base_url: projects/{{project}}/regions/{{region}}/targetTcpProxies
+    collection_url_key: 'items'
+    has_self_link: true
+    input: true
+    description: |
+      Represents a RegionTargetTcpProxy resource, which is used by one or more
+      forwarding rules to route incoming TCP requests to a regional TCP proxy load
+      balancer.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/load-balancing/docs/tcp/internal-proxy'
+      api: 'https://cloud.google.com/compute/docs/reference/rest/beta/targetTcpProxies'
+    min_version: beta
+    async: !ruby/object:Api::OpAsync
+      operation: !ruby/object:Api::OpAsync::Operation
+        kind: 'compute#operation'
+        path: 'name'
+        base_url: 'projects/{{project}}/regions/{{region}}/operations/{{op_id}}'
+        wait_ms: 1000
+      result: !ruby/object:Api::OpAsync::Result
+        path: 'targetLink'
+      status: !ruby/object:Api::OpAsync::Status
+        path: 'status'
+        complete: 'DONE'
+        allowed:
+          - 'PENDING'
+          - 'RUNNING'
+          - 'DONE'
+      error: !ruby/object:Api::OpAsync::Error
+        path: 'error/errors'
+        message: 'message'
+    parameters:
+      - !ruby/object:Api::Type::ResourceRef
+        name: 'region'
+        resource: 'Region'
+        imports: 'name'
+        required: true
+        input: true
+        description: |
+          The region where the regional proxy resides.
+    properties:
+      - !ruby/object:Api::Type::Time
+        name: 'creationTimestamp'
+        description: 'Creation timestamp in RFC3339 text format.'
+        output: true
+      - !ruby/object:Api::Type::String
+        name: 'description'
+        description: 'An optional description of this resource.'
+        input: true
+      - !ruby/object:Api::Type::Integer
+        name: 'id'
```

Comment:
This will conflict with the Terraform `id` field. Should be removed

Reply:
There is an override in `terraform.yml` (this is how it's like in the `compute_region_target_http_proxy` etc):

```
      id: !ruby/object:Overrides::Terraform::PropertyOverride
        name: proxyId
```

Reply:
And this is how it looks in state:
```
          "attributes": {
            "backend_service": "https://www.googleapis.com/compute/v1/projects/my-project/regions/us-central1/backendServices/be-foobar",
            "creation_timestamp": "2022-09-30T04:13:33.235-07:00",
            "description": "Resource created for Terraform acceptance testing",
            **"id": "projects/my-project/regions/us-central1/targetTcpProxies/tp-foobar",**
            "name": "tp-foobar",
            "project": "sfans-hub-project-d647",
            "proxy_bind": false,
            "proxy_header": "NONE",
            **"proxy_id": 8847100439248466354,**
            "region": "us-central1",
            "self_link": "https://www.googleapis.com/compute/v1/projects/my-project/regions/us-central1/targetTcpProxies/tp-foobar",
            "timeouts": null
          },
```

Reply:
I think an easier/clearer way to do this would be adding the proxyId field itself and then specifying `api_name` on it to read from `id` in the API

Reply:
Alright, fixed it.